parsing_scripts/wb_parser.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'account'
+require_relative 'transactions'
+require 'pry'
+require 'watir'
+require 'json'
+require 'json/pure'
+
+class WBParser
+  
+  @@wb_link = 'https://wb.micb.md/'
+  
+  def parse
+    autentificate
+    account = Account.new(browser)
```

Comment:
what if user has more then 1 account?

Reply:
There isn't this requirement in the task